src/ui/find-ui/src/FamilyHubs.ServiceDirectory.Web/Mappers/ServiceMapper.cs

```diff
@@ -21,48 +21,40 @@ private static Service ToViewModel(ServiceDto service)
     {
         Debug.Assert(service.ServiceType == Shared.Enums.ServiceType.FamilyExperience);
 
-        var location = service.Locations.First();
         var eligibility = service.Eligibilities.FirstOrDefault();
 
         var name = service.Name;
-        var contact = service.GetContact();
 
         return new Service(
-            IsFamilyHub(location) ? ServiceType.FamilyHub : ServiceType.Service,
+            service.Id,
+            IsFamilyHub(service.Locations) ? ServiceType.FamilyHub : ServiceType.Service,
             name,
             service.Distance != null ? DistanceConverter.MetersToMiles(service.Distance.Value) : null,
             GetCost(service),
-            location.GetAddress(),
-            service.GetServiceAvailability(),
+            GetWhere(service.Locations),
             GetCategories(service),
-            GetAgeRange(eligibility),
-            contact?.Telephone,
-            contact?.Email,
-            name,
-            GetWebsiteUrl(contact?.Url));
+            GetDeliveryMethods(service.ServiceDeliveries),
+            GetAgeRange(eligibility));
     }
 
+    private static IEnumerable<string> GetWhere(ICollection<LocationDto> locationDtoList)
```

Comment:
GetWhere for what. Maybe something like GetLocations ? Looking at the Service record, is it where the service is located?

Reply:
Resolved: [49701bb](https://github.com/DFE-Digital/fh-services/pull/206/commits/49701bb6987753e61aa0df62d0bbff5f0c29e0b9)